tests/test_history.py

```diff
@@ -1600,7 +1600,8 @@ def test_daily_history_minute_gaps_price_ffill(self, test_name, bar_count):
         # January 5 2015 is the first day, and there is volume only every
         # 10 minutes.
         for day_idx, day in enumerate([pd.Timestamp('2015-01-05', tz='UTC'),
-                                       pd.Timestamp('2015-01-06', tz='UTC')]):
+                                       pd.Timestamp('2015-01-06', tz='UTC'),
+                                       pd.Timestamp('2015-01-12', tz='UTC')]):
```

Comment:
This date is crafted to include a day for which the previous day (when using Timedelta) is not a trading day, correct?
Might want to put a comment here about that, so than any future edits do not lose the intent of the test.

Reply:
Yes - good call.